CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 The intended audience of this file is for py42 consumers -- as such, changes that don't affect
 how a consumer would use the library (e.g. adding unit tests, updating documentation, etc) are not captured here.
 
-## unreleased
+## 1.12.0 - 2021-02-25
 
 ### Added
 
```

Comment:
note should be plural

Reply:
Or have `a` first: 

``sdk.alerts.update_note()` to add or update _a_ note regarding an alert.`

Reply:
to create _an_ `SDKClient`.

Reply:
`mechanism` is a funny word lol

Reply:
🙃